### DISCUSSION TITLE
Chocolatery package structure

### DISCUSSION THREAD

**Discussion by maureto1SICKAG** [chocolatey-community/chocolatey-packages#2322] [General]
Hi there,
I am working on my own Choco repo and came across the following question:
Why are there many packages as normal package and as package.install. Here a dependency in the package refers to the package.install. The actual package only has the XML file and no install of its own.

For example:
There is 7zip and 7zip.install, so 7zip has a dependency on 7zip.install which installs the program.

What is the benefit of this structure?

**Comment by pauby** [comment]
There was a [recent post on Reddit about this](https://www.reddit.com/r/chocolatey/comments/16y691v/why_is_there_often_a_second_version_called/) that may answer your question. This is also answered in our [docs](https://docs.chocolatey.org/en-us/faqs#what-is-the-difference-between-packages-no-suffix-as-compared-to.install.portable).

**Comment by maureto1SICKAG** [comment]
Thank you very much, that answered my question!
